projects/packages/admin-ui/tests/php/bootstrap.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * Bootstrap.
+ *
+ * @package automattic/
+ */
+
+/**
+ * Include the composer autoloader.
+ */
+require_once __DIR__ . '/../../vendor/autoload.php';
+
+define( 'WP_DEBUG', true );
```

Comment:
Why do we need this?

Reply:
When running tests with WorDBless, if WP_DEBUG is not there you can't see the errors. It throws the HTML of that generic WP error page